# goal
string posture_type
bool obstacle
---
# result
bool success
---
# feedback
string current_state
string message
